wiki/Beatmap/Converts/en.md

```diff
@@ -0,0 +1,48 @@
+---
+tags:
+  - convert beatmaps
+  - converted beatmaps
+---
+
+# Converts
+
+![The \"Show converted\" toggle in osu!(lazer)](img/show-converted-button.png "The \"Show converted\" toggle in osu!(lazer)")
+
+In osu! gameplay, **converts** (or *converted beatmaps*) are [beatmaps](/wiki/Beatmap) of a specific [game mode](/wiki/Game_mode) that have been "converted" by the osu! client in-game in order to be playable in another game mode they were not originally created for. All converts are automatically generated based on certain parameters present in the base beatmap.
+
+As it stands, osu! supports conversions of beatmaps from the [osu! game mode](/wiki/Game_mode/osu!) into [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!mania](/wiki/Game_mode/osu!mania).
+
+## Conversions
+
+### osu! to osu!taiko conversion
+
+*See also: [osu!taiko mapping/osu! conversion notes](/wiki/Game_mode/osu!taiko#osu!-conversion-notes)*
+
+Converts between osu! and osu!taiko are generated according to the rhythm, hitsound, and slider velocity changes in the beatmap. Depending on how the [hit objects](/wiki/Gameplay/Hit_object) are being hitsounded, the conversion between the two game modes generally goes as follows:
+
+- Circles, slider heads, slider tails, and slider repeats that bear no additive hitsounds (e.g. no whistle, clap, or finish) are converted into *red notes* (Don).
+- Circles, slider heads, slider tails, and slider repeats that are hitsounded with a whistle, a clap, or both are converted as *blue notes* (Katu).
+- Circles, slider heads, slider tails, and slider repeats that are hitsounded with a finish are converted into *large red notes*.
+- Circles, slider heads, slider tails, and slider repeats that are hitsounded with a finish AND a whistle, a clap, or both are converted into *large blue notes*.
+- Sliders with significant lengths are converted into *drumrolls*.
+- Spinners are converted into *osu!taiko spinners* (dendens/shakers).
```

Comment:
while we don't have an article on shakers, it's the preferred term (and "spinners are converted into osu!taiko spinners" isn't very well-written)
```suggestion
- Spinners are converted into shakers (dendens).
```